webpack/envs/productionConfig.js

```diff
@@ -1,10 +1,10 @@
 // @ts-check
 
 const path = require("path")
-const UglifyJsPlugin = require("uglifyjs-webpack-plugin")
 const WebpackManifestPlugin = require("webpack-manifest-plugin")
 const { HashedModuleIdsPlugin } = require("webpack")
 const { getCSSManifest } = require("../utils/getCSSManifest")
+const TerserPlugin = require("terser-webpack-plugin")
```

Comment:
Terser is included by default with webpack.

Reply:
Meaning, `mode: "production"`? If so can we omit the `optimization` key completely?